internal/querycoord/querynode.go

```diff
@@ -255,11 +257,12 @@ func (qn *queryNode) releasePartitionsInfo(collectionID UniqueID, partitionIDs [
 			}
 		}
 		info.PartitionIDs = newPartitionIDs
-		err := removeNodeCollectionInfo(collectionID, qn.id, qn.kvClient)
+		err := saveNodeCollectionInfo(collectionID, info, qn.id, qn.kvClient)
```

Comment:
Will there be any side effect after all partitions of a collection is released?

Reply:
wrong use of removeNodeCollectionInfo